### DISCUSSION TITLE
Please add amount in BTC into the invoice object

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4404] [Ideas / Feature Requests]
Currenty the invoice object that is received from Greenfield API only provides info regarding the total amount in USD, however we need to also know what was the deposited amount in BTC. Or maybe there is some other way to query such info via API?

**Comment by mp21-org** [comment]
Please use https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoicePaymentMethods to obtain all payments of an invoice and add the totalPaid values (an invoice can be paid multiple times, with different crypto currencies).

**Comment by ilearnio** [comment]
@petzsch Thank you! It helped!
